ci/kind/kind-setup.sh

```diff
@@ -316,7 +316,7 @@ EOF
     eval "$cmd $(get_encap_mode) | kubectl apply --context kind-$CLUSTER_NAME -f -"
 
     if [[ $PROMETHEUS == true ]]; then
-      kubectl apply --context kind-$CLUSTER_NAME -f $THIS_DIR/../../build/yamls/antrea-prometheus-rbac.yml
+      kubectl apply --context kind-$CLUSTER_NAME -f $THIS_DIR/../../build/yamls/antrea-prometheus.yml
```

Comment:
github actions also call this function, so don't change it. Instead, add code in test.sh if you need.